k8s/demo/cluster-01-overlay/nfdiv/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
this needs to be referenced somewhere

Reply:
I am not sure where we need to reference  `k8s/demo/cluster-01-overlay/nfdiv/kustomization.yaml` could you please point me to the config ?

Reply:
https://github.com/hmcts/cnp-flux-config/blob/master/k8s/demo/cluster-01-overlay/kustomization.yaml

Reply:
thanks @timja